instance/models/openedx_appserver.py

```diff
@@ -362,18 +349,28 @@ def smtp_relay_settings(self):
             return None
 
     @property
-    def organization_users(self):
+    def admin_users(self):
         """
-        :return: Github usernames of the organization this instance owned by.
+        Return GitHub usernames of admin users for this instance.
+
+        Admins are all users with `is_superuser` set to `True`,
+        as well as all members of the organization that owns this instance.
         """
         users = []
+        # Add members of owning organization to the list of users that can administer this instance
         organization = self.instance.ref.owner
         if organization:
             usernames = UserProfile.objects.filter(
                 organization=organization
             ).values_list('github_username', flat=True)
             users += usernames if usernames else []
 
+        # Add superusers to the list of users that can administer this instance
+        usernames = UserProfile.objects.filter(
```

Comment:
This will hit the database twice for the same query but for different filtering, we can reduce the hits to one if we have on QuerySet and we iterate over it once. Something like the following would be more efficient: (Didn't test it though)
```PYTHON
users_qs = UserProfile.objects.filter(user__is_superuser=True)

organization = self.instance.ref.owner
if organization:
    users_qs = users_qs.filter(organization=organization)

usernames = list(users_qs.values_list('github_username', flat=True))
```

Reply:
I didn't read all the code, but the people we want in a sandbox are: everyone from the organization *plus* all of our superusers.

Reply:
@taranjeet That's true. I didn't change the end result in my suggestion, I only changed the approach.

Reply:
But we don't require people in a sandbox organization to have a superuser account in our Ocim. So @ahmedaljazzar I think that the code you proposed with 1 query won't work. It's not a problem to have 2 queries either.

Reply:
True, @clemente. A thing we can do here is to use [Q objects](https://docs.djangoproject.com/en/1.10/topics/db/queries/#complex-lookups-with-q-objects).
```Python
users_qs = UserProfile.objects.filter(Q(user__is_superuser=True) | Q(organization=organization))
```
However, it's up to @taranjeet to use it or not.

Reply:
Keeping the current code, hence marking this as resolved.